test/cmd/admin.sh

```diff
@@ -299,3 +299,8 @@ os::cmd::expect_success_and_not_text "oc get clusterrolebindings/cluster-admins
 os::cmd::expect_success_and_not_text "oc get rolebindings/cluster-admin         --output-version=v1 --template='{{.subjects}}' -n default" 'cascaded-group'
 os::cmd::expect_success_and_not_text "oc get scc/restricted                     --output-version=v1 --template='{{.groups}}'"              'cascaded-group'
 echo "user-group-cascade: ok"
+
+
+os::cmd::expect_failure_and_text 'openshift ex diagnostics FakeDiagnostic --diagnostics AlsoMissing' 'No requested diagnostics are available: requested=AlsoMissing, FakeDiagnostic'
+os::cmd::expect_failure_and_text 'openshift ex diagnostics AnalyzeLogs --diagnostics AlsoMissing' 'Not all requested diagnostics are available: requested=AlsoMissing, AnalyzeLogs'
+echo "diagnostics: ok"
```

Comment:
FWIW there is a file for diagnostics test cmds once this merges https://github.com/openshift/origin/pull/6397/files#diff-6061a923ddc8d6dfdd6620834faccb34